modules/nf-core/ampcombi/main.nf

```diff
@@ -0,0 +1,45 @@
+process AMPCOMBI {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::ampcombi=0.1.3" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/ampcombi:0.1.3--pyhdfd78af_0':
+        'quay.io/biocontainers/ampcombi:0.1.3--pyhdfd78af_0' }"
+
+    input:
+    tuple val(meta),  path(amp_input)
+    path(faa_folder)
+    val(outdir)
+
+    output:
+    tuple val(meta), path("${outdir}")          , emit: results_dir
+    path("${outdir}/*/*_diamond_matches.txt")   , emit: txt
+    path("${outdir}/*/*_ampcombi.csv")          , emit: csv
+    path("${outdir}/*/*_amp.faa")               , emit: faa
+    path("${outdir}/*.log")                     , optional:true, emit: log
+    path("${outdir}/*")                         , emit: results_subdir
+    path("${outdir}/amp_ref_database")          , optional:true, emit: results_subdir_db
```

Comment:
Some of these should maybe also output a tuple with with `meta` map so that one can associate the result with the sample metainformation.

Reply:
But given this is aggregating a lot of input from different samples to aggregate, does it make sense to include meta? There is no actual consistency between files (i.e., shared DBs etc.)

Reply:
Meta is related to the sample-name, right? Actually, we should include it to the results_subdir, which is created per sample.

Reply:
But a (typical) meta map has only a single ID, how would you assign meta to each sample separately?

Reply:
The way I understood the latest use is that the process will be called per sample rather than for all samples together.